RockWeb/App_Code/GetChannelFeed.cs

```diff
@@ -172,11 +172,11 @@ public void ProcessRequest( HttpContext context )
             {
                 if ( channel.ContentChannelType.IncludeTime )
                 {
-                    content = content.Where( c => c.ExpireDateTime >= RockDateTime.Now );
+                    content = content.Where( c => !c.ExpireDateTime.HasValue || c.ExpireDateTime >= RockDateTime.Now );
```

Comment:
This looks like a core thing? How often does this change? I'm just thinking through things like merge conflicts whenever we pull from `pre-alpha-release`. Maybe this could get PR'd to core?

Reply:
FWIW, a PR to core could come after we merge this. It wouldn't have to happen before I would say this is good.

Reply:
As we discussed, this was copied from a core change that will be coming later (actually it's probably in that latest pre-alpha).